### ISSUE TITLE
Aggravated by menstruation

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#10425]
**Preferred term label:**

**Synonyms**
Aggravated by period
Aggravated by monthly period
Aggravated by monthly menses

**Definition (free text, please give PubMed ID)**
PMID: [32966006] Muscle stiffness observed in Myotonia Congenita is often ameliorated by exercise or repetitive movement. This effect is called the “warm-up” phenomenon (though it is quickly lost on cessation of activity). Men appear to be more severely affected by MC than women. In addition, symptoms often **worsen during pregnancy and menstruation**; these observations imply that sex hormones affect CIC-1 channel function.

PMID: 30523495 Environmental and other triggering factors are suggested to cause the onset and the clinical relapses of Behçet's syndrome (BS), a multi-systemic inflammatory disorder.(...) Stress is a common self-triggering factor for most BS patients. Stimuli such as some foods can activate oral ulcers, and may be linked to the histamine content of the food. Oral/skin trauma and **menstruation** associated with hormonal factors **aggravate**, whereas allergy/atopy seem to alleviate the symptoms of BS.

**Parent term (use hpo.jax.org/app)**
Aggravated by HP:0025285

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
(Myotonia Congenita) Thomsen and Becker disease  ORPHA:614
Behçet's syndrome  ORPHA:117 

**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
@carolina-f 
added as HP:6000699
